tests/mir-opt/dest-prop/simple.rs

```diff
@@ -1,9 +1,12 @@
-// skip-filecheck
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
 //! Copy of `nrvo-simple.rs`, to ensure that full dest-prop handles it too.
 //@ unit-test: DestinationPropagation
 // EMIT_MIR simple.nrvo.DestinationPropagation.diff
 fn nrvo(init: fn(&mut [u8; 1024])) -> [u8; 1024] {
+    // CHECK-LABEL: fn nrvo(
+    // CHECK: debug init => [[init:_.*]];
+    // CHECK-NOT: {{_.*}} = [[init]];
+    // CHECK: move [[init]](move {{_.*}})
```

Comment:
IIUC, the point of this test is to track what happens to `buf`.

Reply:
I missed that, because there were no changes to `buf`. I added some more checks, let me know if it's better.